### PR TITLE
Generating localizable strings

### PR DESCRIPTION
**Pull request by stefanceriu**

### CODE REVIEW

**Comment by stefanceriu** [comment]
I'm wondering if we should expose methods instead of format strings from the `StringsLocalizable` protocols. 
Also, couldn't find translations for:
* auth_username_in_use
* auth_invalid_user_name
* auth_reset_password_error_unauthorized
* auth_reset_password_error_not_found

File: MatrixKit.xcodeproj/xcshareddata/xcschemes/MatrixKitSample.xcscheme [hunk 1]
```diff
@@ -40,7 +40,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
+      shouldUseLaunchSchemeArgsEnv = "YES"
+      language = "de">
```
[the remark this answers is not in the document]
Comment (stefanceriu) [reply MatrixKit.xcodeproj/xcshareddata/xcschemes/MatrixKitSample.xcscheme:44]: Ah yes, sorry about that 😅

**Review by pixlwave — COMMENTED** [review]
Ignore my comment on `copy`, I twigged looking at the call site 😄

Seeing those `formatTimeD`/`formatTimeH`/`formatTimeM`/`formatTimeS` strings was surprising. Unrelated to this PR, we should really look into whether using an NSFormatter there would work.

File: MatrixKit/Views/DeviceView/MXKDeviceView.m [hunk 2]
```diff
@@ -147,32 +149,33 @@ - (instancetype)initWithDevice:(MXDevice*)device andMatrixSession:(MXSession*)se
...
                                                              attributes:@{NSForegroundColorAttributeName : _defaultTextColor,
                                                                           NSFontAttributeName: [UIFont systemFontOfSize:14]}]];
             [deviceInformationString appendAttributedString:[MXKDeviceView verticalWhitespace]];
             
             [deviceInformationString appendAttributedString:[[NSMutableAttributedString alloc]
-                                                      initWithString:[NSBundle mxk_localizedStringForKey:@"device_details_identifier"]                                                      attributes:@{NSForegroundColorAttributeName : _defaultTextColor,
-                                                                   NSFontAttributeName: [UIFont boldSystemFontOfSize:14]}]];
+                                                             initWithString:[MatrixKitL10n deviceDetailsIdentifier]
+                                                             attributes:@{NSForegroundColorAttributeName : _defaultTextColor,
```
[the remark this answers is not in the document]
Comment (stefanceriu) [reply MatrixKit/Views/DeviceView/MXKDeviceView.m:169]: Yes, it's already there but indented way to the right.

[the remark this answers is not in the document]
Comment (pixlwave) [reply MatrixKit/Views/DeviceView/MXKDeviceView.m:169]: Oh yeah! 🤦‍♂️

**Review by pixlwave — APPROVED** [review]

**Pull request merged**